File: 1352A. Sum of Round Numbers.js
function getNonZeroDigits(str) {
	var count = 0
	for (var ch of str)
		if (ch !== "0")
			++count
	return count
}

function getAllRoundNumbers(str) {
	var len = str.length
	for (var i in str)  {
		if (str[i] !== "0") {
			print(((str[i]-'0')*Math.pow(10,len-i-1)), end=" ")
		}
	}
}

var tc = Number(readline())
while(tc--) {
	var str = readline()
	print(getNonZeroDigits(str))
	getAllRoundNumbers(str)
}

File: 1352A. Sum of Round Numbers.py
def getNonZeroDigits(str):
	count = 0
	for ch in str:
		if ch != "0":
			count = count+1
	return count

def getAllRoundNumbers(str):
	l = len(str)
	for i in range (0,l):
		if str[i] != "0":
			print(int(str[i])*pow(10,l-i-1),end=' ')
	print()
	return

tc = int(input())
for pos in range(tc):
	str = input()
	print(getNonZeroDigits(str))
	getAllRoundNumbers(str)